src/briefcase/exceptions.py

```diff
@@ -197,41 +200,33 @@ def __init__(self, app_bundle_path):
         super().__init__(f"Unable to find {str(app_bundle_path / 'briefcase.toml')!r}")
 
 
-class MissingSupportPackage(BriefcaseCommandError):
-    def __init__(self, python_version_tag, platform, host_arch, is_32bit):
-        self.python_version_tag = python_version_tag
-        self.platform = platform
-        self.host_arch = host_arch
-        self.is_32bit = is_32bit
+class MissingPlatformSupport(BriefcaseCommandError):
+    def __init__(self, python_version_tag, platform, host_arch, is_32bit, item):
```

Comment:
The exception should be retaining the properties that constructed it so it can be interrogated later.
```suggestion
    def __init__(self, python_version_tag, platform, host_arch, is_32bit, item):
        self.python_version_tag = python_version_tag
        self.platform = platform
        self.host_arch = host_arch
        self.is_32bit = is_32bit
```